src/deployment/deploy.py

```diff
@@ -72,7 +72,8 @@
     update_pool_registration,
 )
 
-USER_IMPERSONATION = "311a71cc-e848-46a1-bdf8-97ff7156d8e6"
+USER_READ_PERMISSION = "e1fe6dd8-ba31-4d61-89e7-88639da4683d"
```

Comment:
For each of these constants, are there docs we can link to?

Reply:
could not find an external doc with all the guids for the permissions. So I added a comment explaining how I got those values

Reply:
Can we link to this document, which demonstrates it?

https://docs.microsoft.com/en-us/azure/aks/azure-ad-integration-cli#create-azure-ad-server-component